src/service/url-replacements-impl.js

```diff
@@ -397,6 +397,37 @@ export class GlobalVariableSource extends VariableSource {
       return nav.userAgent;
     });
 
+    const HTML_ATTR_MAX_RETURN_SIZE = 10;
+    this.set('HTML_ATTR', (doc, cssSelector, ...attributeNames) => {
```

Comment:
^^ Is this allowed?

Reply:
the `doc` object should be from the service itself. I guess the problem is we're right now using a global urlreplacement service.

Reply:
a workaround is to move this binding into a4a code that has access to the right doc object, and pass the binding via the `dynamicBindings` param

Reply:
Correct re: global service. So its doc is the pub page, and we want to query the creative.
I like your workaround idea. Will do. Thanks!